### ISSUE TITLE
tables produced by xinha do not have headers and fail to meet WAI guidelines. (Trac #640)

### ISSUE THREAD

**Issue by sleemanj** [sleemanj/xinha#640]
The WAI Accessibility guidelines state that tables should use the TH tag to specify a header. Would it be possible to add something to Xinha to ask the user for the table headings or simply take the top row as headings. 

_Reported by anonymous, migrated from http://trac.xinha.org/ticket/640_
